### DISCUSSION TITLE
create dag with DAG Serialization mode.

### DISCUSSION THREAD

**Discussion by diablowu** [apache/airflow#15155] [Q&A]
I had enabled DAG Serialization(https://airflow.apache.org/docs/apache-airflow/stable/dag-serialization.html), but how should I create a new DAG file?I added the dag file in ${AIRFLOW_HOME}/dags, but it does not appear in the UI.

**Comment by turbaszek** [comment]
@diablowu what Airflow version do you use? In 2.0 serialisation is turned on by default and is required.
